tests/functional/remote/04-symlink-dirs.t

```diff
@@ -37,14 +40,16 @@ create_test_global_config "" "
             share/cycle = \$TMPDIR/\$USER/cylctb_tmp_share_dir
             work = \$TMPDIR/\$USER
         [[[$CYLC_TEST_INSTALL_TARGET]]]
-            run = \$TMPDIR/\$USER/test_cylc_symlink/ctb_tmp_run_dir
```

Comment:
`$TMPDIR/$USER` does not exist on our new machine and `$TMPDIR` path changes from node to node :/

Switched to a permanent directory `$HOME/cylctb-symlinks` with one subdir for each test.